README.md

```diff
@@ -62,21 +62,35 @@ to run that script, then refresh your browser.
     - Yellow lines contain expressions, some of which were visited, but others not.
     - White lines are those that don't contain visitable expressions. They may have type declarations, keywords, or something else that Bisect_ppx did not, or cannot instrument.
 
-You can submit a coverage report to Coveralls.io using [ocveralls][ocveralls].
+See also the [advanced usage][advanced].
+
+### Coveralls.io
+
+You can submit a coverage report to Coveralls.io using [ocveralls][ocveralls] or
+(**New**) you can generate a Coveralls json report using the `bisect-ppx-report`
+tool with the `-coveralls` flag.
 Note that Bisect_ppx reports are more precise than Coveralls, which only
-considers whole lines as visited or not.
+considers whole lines as visited or not. The built in Coveralls reporter will
+consider a full line unvisited if any point on that line is not visited,
+check the html report to verify precisly which points are not covered.
 
-See also the [advanced usage][advanced].
+Example using the built in Coveralls reporter:
+
+      bisect-ppx-report \
+          -I _build/ \
+          -coveralls coverage.json \
+          -service-name travis-ci \
+          -service-job-id 1234567 \
```

Comment:
If using Travis as the example, should this perhaps be `$TRAVIS_JOB_ID` or another [environment variable](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables)?

Reply:
Maybe we can make a new doc page to cover the environment variables / other usages?

Reply:
I updated to reference the variable for now, which may be sufficient. It may be nice however to document the environment variables that ocveralls supported:
https://github.com/sagotch/ocveralls/blob/master/src/ocverallsCI.ml#L26-L30